client-sdk/ts-web/rt/src/address.ts

```diff
@@ -23,3 +43,11 @@ export async function fromMultisigConfig(config: types.MultisigConfig) {
         configU8,
     );
 }
+
+export function toBech32(addr: Uint8Array) {
+    return oasis.address.toBech32(ADDRESS_PREFIX, addr);
```

Comment:
these can call oasis.staking.addressToBech32 and ...From... similarly below, to use the `oasis` prefix